docker/development/eth-node/package.json

```diff
@@ -10,7 +10,7 @@
   },
   "dependencies": {
     "@nomiclabs/hardhat-ethers": "2.2.3",
-    "@unlock-protocol/hardhat-plugin": "0.0.18",
+    "@unlock-protocol/hardhat-plugin": "latest",
```

Comment:
it's good to use `latest` here so we make sure we are up to date in our test pipeline. Maybe that requires some tweak in renovate config to prevent it from opening PR ?

Reply:
HA! The problem with "latest" is that it means we can have inconsistent builds because the `yarn.lock` file will be outdated?